src/react-ruler/types.ts

```diff
@@ -157,4 +157,20 @@ export interface RulerProps {
    * portal container for other frameworks
    */
   portalContainer?: HTMLElement;
+  /**
+   * color of theme
+   * @default dark
+   */
```

Comment:
Dmytro and I deleted doc module.
So I think adding comments such as this is not useful since we no longer support this.